### PR TITLE
Add Dropbox

### PR DESCRIPTION
**Pull request by vincecima**

### CODE REVIEW

File: just-install.json [hunk 1]
```diff
@@ -157,6 +157,20 @@
...
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/S"
+          ],
+          "extension": ".exe"
+        },
+        "x86": "https://www.dropbox.com/download?type=full&build={{.version}}&plat=win"
+      },
+      "version": "3.12.6"
```
Comment (lvillani) [inline just-install.json]: I would remove the `build=...` parameter from the URL and change the "version" field to "latest", this way users always get the latest version.

Comment (vincecima) [inline just-install.json]: :+1: should we prefer the latest link vs pinning to a version?

Comment (lvillani) [inline just-install.json]: Yes, that lightens the maintenance burden. We should pin to a specific version only if two versions can live side by side or there's widespread reported breakage with a particular version.

I'll update the contributing guidelines ASAP.

**Comment by vincecima** [comment]
@lvillani all set and I squashed the extra commit

**Comment by lvillani** [comment]
Great! Thanks!

**Pull request merged**